src/main/java/UtilsTest/PasswordsTest.java

```diff
@@ -0,0 +1,66 @@
+package UtilsTest;
+
+import Utils.Passwords;
+import Utils.Tuple;
+import jdk.nashorn.internal.ir.annotations.Ignore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PasswordsTest {
+
+    String password;
+
+    @BeforeEach
+    void setUp() {
+        password = "Password123";
+    }
+
+    @AfterEach
```

Comment:
The `@AfterEach` annotated function will run after every test in the class. This will be used to revert any permanent changes to the database or close any `Writers `we have running.